examples/solace_dmr.yml

```diff
@@ -0,0 +1,74 @@
+- name: Playbook to add a DMR Cluster named 'foo'
+  hosts: localhost
+  tasks:
+  - name: Remove 'foo' DMR Cluster
+    solace_dmr:
+      name: foo
+      state: absent
+  - name: Add 'foo' DMR Cluster
+    solace_dmr:
+      name: foo
+      state: present
+      settings:
+        tlsServerCertMaxChainDepth: 6
+        authenticationBasicPassword: Solace123!
```

Comment:
I see you removed passwords in a previous commit - here's one more

Reply:
I just use this as a random password. I did remove the parameter override for the Solace admin password so it's easier for a user who set up a docker container with the default admin user/pw to run the ansible tasks.
Happy to replace this with another placeholder that you prefer - you used "secret_password" in the queue example - shall I use that?